infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -175,6 +177,10 @@ function ensure_staging_project() {
     color 6 "Ensuring ${cip_principal} can view vulnernability scanning results for project: ${project}"
     ensure_project_role_binding "${project}" "${cip_principal}" "roles/containeranalysis.occurrences.viewer"
 
+    # Ensure staging AR repo
+    color 3 "Ensuring staging AR repo: ${location}-docker.pkg.dev/${project}/images"
+    ensure_ar_repo "${project}" "${location}" 2>&1 | indent
```

Comment:
Can you grant the `roles/artifactregistry.repoAdmin` role to the writers group?

Reply:
Will do in a follow-up PR. Feel free to open one if the binding is urgently needed.